appveyor.yml

```diff
@@ -55,5 +55,5 @@ test_script:
 - ps:  src\bench_bitcoin.exe -evals=1 -scaling=0
 - ps:  python test\util\bitcoin-util-test.py
 - cmd: python test\util\rpcauth-test.py
-- cmd: python test\functional\test_runner.py --force --quiet --combinedlogslen=4000 --exclude "feature_notifications,wallet_multiwallet,wallet_multiwallet.py --usecli"
+- cmd: python test\functional\test_runner.py --force --quiet --combinedlogslen=4000 --exclude "feature_notifications,wallet_multiwallet"
```

Comment:
As you touch this file anyway, mind to move it to `.appveyor.yml` as per #14306, if you agree with the rationale? Personally I don't care, but the other pull request will just conflict all the time and is probably too noisy keeping around.